src/vmm/src/guest_config/templates/mod.rs

```diff
@@ -131,11 +320,124 @@ where
 
 #[cfg(test)]
 mod tests {
+    #[cfg(target_arch = "x86_64")]
```

Comment:
Can possibly group together x86_64 and aarch64 stuff to avoid duplicating these for every line.

Reply:
I do have some tweaks downstream that aggregates this a bit better.

Reply:
Yes, the changes are quite big, but yield more benefit when there are multiple crates involved.